dask/tests/test_parquet.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+import pandas as pd
+import pytest
+from dask import dataframe as dd
+
+'''
+@pytest.fixture
+def gzip_parquet_file(tmpdir):
+
+
+    return parquet_file_path
+'''
+
+
+@pytest.mark.parametrize('compression, engine', [
+    (None, 'pyarrow'), (None, 'fastparquet'),
+    ('SNAPPY', 'pyarrow'), ('SNAPPY', 'fastparquet'),
+    ('GZIP', 'pyarrow'), ('GZIP', 'fastparquet')])
+def test_writing_parquet_with_gzip_compression(compression, engine):
```

Comment:
We already have a test file for this in `dask/dataframe/io/tests/test_parquet.py`. Please move this test there.

Reply:
Also note that in that file there is a fixture for `engine` that skips tests requiring that engine if it's not installed. There's also a fixture for handling temporary directories. Your test would then look something like (not tested):

```python
@pytest.mark.parametrize('compression', ['default', None, 'snappy', 'gzip'])
def test_to_parquet_compression(tmpdir, engine, compression):
    fn = str(tmpdir)
    if engine == 'fastparquet' and compression == 'snappy':
        pytest.importorskip('snappy')
    df = pd.DataFrame({'x': ['a', 'b', 'c'] * 10, 'y': [1, 2, 3] * 10})
    ddf = dd.from_pandas(df, npartitions=3)
    ddf.to_parquet(fn, engine=engine, compression=compression)
    ddf2 = dd.read_parquet(fn, engine=engine)
    assert_eq(ddf2, df)
```

Also note that `snappy` isn't a required dependency of fastparquet, and may not be installed. `pyarrow` bundles `libsnappy`, so there's no fear of it missing there. I'd either check if snappy is installed and skip if not (as done above), or just not test with `snappy`. Up to you.